packages/react/src/util/elementOverflow.ts

```diff
@@ -1,9 +1,47 @@
-export function elementOverflow(element: HTMLElement, container: HTMLElement) {
+// Default for `getContainer`, meant for use in suggestion menus. Traverses
+// ancestors of the passed element until a suggestion menu root is found.
+const defaultGetContainer = (element: HTMLElement) => {
+  let container: HTMLElement = element;
+
+  while (
```

Comment:
I think you can just use https://developer.mozilla.org/en-US/docs/Web/API/Element/closest